app/views/users/check_email.jbuilder

```diff
@@ -1,3 +1,3 @@
 # frozen_string_literal: true
 
-json.email_exists @user.present?
+json.emailExists @user.present?
```

Comment:
Why did you change it? It's not a ruby way :) Use snake case here, please :)

Reply:
Sergey said that it's better for front-end team.